Mathlib/Order/CompletePartialOrder.lean

```diff
@@ -82,3 +82,6 @@ end CompletePartialOrder
 instance CompleteLattice.toCompletePartialOrder [CompleteLattice α] : CompletePartialOrder α where
   sSup := sSup
   lubOfDirected _ _ := isLUB_sSup _
+
+lemma sSup_eq_bot' [CompleteLattice α] {s : Set α} : sSup s = ⊥ ↔ s = ∅ ∨ s = {⊥} := by
```

Comment:
It seems like this lemma belongs in the same file as `sSup_eq_bot`. What do you think?

Reply:
I agree, this seems more correct.